crates/trans_token/src/storage.rs

```diff
@@ -240,6 +240,13 @@ where
     if amount.is_zero() {
         return Ok(());
     }
+
```

Comment:
let's update the comment for this fn

Reply:
done in #[4c5a9f3](https://github.com/anoma/namada/pull/3675/commits/4c5a9f35c879efafcaad87b9e2d580ed834d183b)